src/binomial.jl

```diff
@@ -131,6 +133,13 @@ function ci_wilson(x::BinomialTest, alpha::Float64=0.05)
     (μ-q*σ, μ+q*σ)
 end
 
+# Arcsine transformation interval
+function ci_arcsine(x::BinomialTest, alpha::Float64=0.05)
+    q = quantile(Normal(), 1-alpha/2)
+    p = x.x / x.n
+    σ = q/(2*sqrt(x.n))
```

Comment:
If you feel up to it, I'd call this `z` since it's neither a standard deviation nor a standard error. Matching the notation from Wikipedia (and maybe even linking to the article in a comment) would help people understand this code if they read it at a future date and want to understand how it works: https://en.wikipedia.org/wiki/Binomial_proportion_confidence_interval#Arcsine_transformation